engine/graphics/src/graphics.h

```diff
@@ -723,11 +723,20 @@ namespace dmGraphics
     uint8_t     GetNumTextureHandles(HTexture texture);
     uint32_t    GetTextureUsageHintFlags(HTexture texture);
 
+    /**
+     * Get status of texture.
+     *
+     * @name GetTextureStatusFlags
+     * @param texture HTexture
+     * @return  TextureStatusFlags enumerated status bit flags
+     */
+    uint32_t    GetTextureStatusFlags(HTexture texture);
+    void        EnableTexture(HContext context, uint32_t unit, uint8_t id_index, HTexture texture);
+    void        DisableTexture(HContext context, uint32_t unit, HTexture texture);
```

Comment:
Just moved these functions to the same place as the other texture functions.

Q: This file isn't consistent in terms of documentation, should we consolidate the API docstrings here?